ci/test/resources/build.yaml

```diff
@@ -3,6 +3,13 @@ steps:
    name: default_ns
    namespaceName: default
    public: true
+ - kind: buildImage
+   name: inline_image
```

Comment:
This image is for testing inline image, yes?  (Otherwise I'd want a more semantic name.)

Reply:
Yes